app/polls/onboarding/loading-insights/components/LoadingInsightsPage.js

```diff
@@ -0,0 +1,73 @@
+'use client'
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+import LoadingFooter from '../../components/LoadingFooter'
+import LoadingList from '../../components/LoadingList'
+
+export default function LoadingInsightsPage({ pathname }) {
+
+  const router = useRouter()
+  const [loadingItems, setLoadingItems] = useState([
+    {
+      label: 'Gathering your constituents',
+      status: 'loading',
+    },
+    {
+      label: 'Establishing community age groups',
+      status: 'pending',
+    },
+    {
+      label: 'Identifying homeowner vs. renter ratios',
+      status: 'pending',
+    },
+    {
+      label: 'Looking for families with children under 18',
+      status: 'pending',
+    },
+    {
+      label: 'Analyzing education levels',
+      status: 'pending',
+    },
+    {
+      label: 'Mapping income brackets',
+      status: 'pending',
+    },
+    {
+      label: 'Capturing community diversity',
+      status: 'pending',
+    },
+    {
+      label: 'Generating constituency profile',
+      status: 'pending',
+    },
+  ])
+
+  // Navigate to insights when all steps are complete
+  const onComplete = () => {
+    router.replace('/polls/onboarding')
+  }
+
+  return (
+    <div className="fixed inset-x-0 bottom-0 top-14 flex flex-col bg-white md:bg-muted">
```

Comment:
Why are you using `fixed` here? On smaller screens, you won't be able to scroll. I see you added overflow-y-auto below, but why not just use the default browser scrolling?